release-engineering/packaging.md

```diff
@@ -0,0 +1,125 @@
+
+## Communication
+
+The [Patch Release Team][patch-release-team] (patch releases) or [Branch Manager][branch-manager-handbook] (minor releases) will reach out to the [Kubernetes Build Admins][kubernetes-build-admins] via [email][kubernetes-build-admins] or `#release-managers` on [Kubernetes Slack](https://slack.k8s.io) (for more synchronous communication) requesting help to build the debs and rpms.
+
+Patch Release Team members or Branch Managers requesting debs/rpms should be sure to provide explicit details on the package name(s), version(s), and revision(s) they need built.
+
+**n.b. As much as possible, communications with the Kubernetes Build Admins should happen on public forums (email, Slack public channels), not direct messaging, except in instances where doing so would run contrary to our [security policies][security-release-process] e.g., building packages for a release which addresses an embargoed CVE.**
+
+## Release Steps
+
+In this process, we are pulling the artifacts published by a Release Manager to the GCS bucket and building debs/rpms to be published to the rapture repository. 
+
+Refer to the [Branch Manager handbook][branch-manager-build-and-release] for details on the artifacts that are built by the Branch Manager.
+
+See also the [rapture documentation][rapture-readme], which contains details on running `rapture` (Google internal packaging tool).
+
+### Permissions
+
+Must be a member of mdb group - [mdb/cloud-kubernetes-release](mdb/cloud-kubernetes-release) to be able to perform the debs/rpms releases.
+
+
+### Clone Release Repository
+
+Before proceeding with the release, ensure the [kubernetes/release][kubernetes/release] repository is checked out. 
+
+**NOTE**: It is a good idea to start in a clean directory, when possible.
+
+_The directory name can be anything. We'll use `$HOME/k8s-1.20.0` as the example here._
+
+```shell
+mkdir -p $HOME/k8s-1.20.0 
+cd $HOME/k8s-1.20.0
+git clone https://github.com/kubernetes/release.git
+cd release
+```
+
+### Authenticate
+
+Run the following commands to ensure that we are logged in and also the proper project context is setup.
+
+
+```
+prodaccess
+gcloud auth login
+gcloud config set project kubernetes-release-dev
+```
+
+### Build the Debs & RPMs
+
+The entire build process takes several hours. Once you are ready to begin, the debs and rpms can be built using [rapture][rapture]. 
+
+`rapture` can be executed as follows:
+
+```shell
+/google/src/head/depot/google3/experimental/users/mehdy/kubernetes/k8s-rapture.sh <version> # <version> should be the Kubernetes version we are building the debs/rpms for e.g., `1.20.0`
+```
+
+#### Notes
+- There are several points during the process where you will be prompted to answer “y/N” or your password.
+- There will be a warning about trusty. This can be ignored. 
```

Comment:
What is trusty?  Why is it warned?  Why can it be ignored?  Is there a bug number tracking resolving this false warning?